cas-server-integration-pac4j-core/src/main/java/org/jasig/cas/authentication/handler/support/AbstractPac4jAuthenticationHandler.java

```diff
@@ -18,6 +19,7 @@
  * @author Jerome Leleu
  * @since 4.1.0
  */
+@Component("abstractPac4jAuthenticationHandler")
```

Comment:
OK. But you won't set a value if you don't have an implementation, so the child class will also have a `@Component`. It feels useless to me, doesn't it?